lib/lita-reviewme/github.rb

```diff
@@ -0,0 +1,66 @@
+require 'delegate'
+
+module Lita
+  module Reviewme
+    class Github
```

Comment:
Good point! I'll add some specs and use that to facilitate the refactors you've suggested.